app/views/question_flow/how_to_aggregate.html.slim

```diff
@@ -0,0 +1,31 @@
+- content_for :page_title
+  = t(".title")
+- content_for :back
+  = back_link(:how_to_aggregate, @check, @back_buttons_invoke_browser_back_behaviour)
+
+.govuk-grid-column-full
+  = render "shared/heading"
+  .govuk-panel.panel-blue.govuk-panel--confirmation class="govuk-!-text-align-left govuk-!-margin-bottom-9"
+    .govuk-panel__body
+      h1.govuk-heading-l = t(".title")
+      p.govuk-body = t(".panel_text")
+  h2.govuk-heading-s = t(".what_happens_next")
+  - t(".paragraphs_1").each
+    p.govuk-body = _1
+  ul.govuk-list.govuk-list--bullet
+    - t(".bullets_1").each
+      li = _1
+  p.govuk-body
+    = t(".paragraph_2_html",
+        link: link_to(t(".link_text"), ExternalLinkService.call(document: :legislation_CLAR_2013_individual_resources)))
```

Comment:
just thinking on this, because we are calling the service directly rather than going via the documents controller, we don't track an analytics event here. Maybe that's fine, just a small point.

Reply:
You're absolutely right, I forgot how the link system worked! Will fix.